.github/workflows/test.yml

```diff
@@ -29,8 +29,8 @@ on:
       cylc_rose_repo:
         description: The cylc-rose repo to test against
         required: false
-  schedule:
-  - cron: '37 04 * * 1-5' # 03:37, Monday-Friday
+  # schedule:
+  # - cron: '37 04 * * 1-5' # 04:37, Monday-Friday
```

Comment:
?

Reply:
Cherry-pick of https://github.com/metomi/rose/pull/2850 [stop scheduled tests as they're consistently failing](https://github.com/metomi/rose/pull/2850/commits/9341948655a6a915be50f92e7cdb56c78421a059)